### PR TITLE
[Azure Data Factory] Generate new SDK

### PR DESCRIPTION
**Pull request by meiyan-Li**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by meiyan-Li** [comment]
Related spec PRs:
https://github.com/Azure/azure-rest-api-specs/pull/4424
https://github.com/Azure/azure-rest-api-specs/pull/3957
https://github.com/Azure/azure-rest-api-specs/pull/4747
https://github.com/Azure/azure-rest-api-specs/pull/4780
https://github.com/Azure/azure-rest-api-specs/pull/4724
https://github.com/Azure/azure-rest-api-specs/pull/4939
https://github.com/Azure/azure-rest-api-specs/pull/5007
https://github.com/Azure/azure-rest-api-specs/pull/5031

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/DataFactory/Management.DataFactory/Generated/Models/StoredProcedureParameter.cs [hunk 1]
```diff
@@ -61,18 +60,5 @@ public StoredProcedureParameter(object value, string type = default(string))
         [JsonProperty(PropertyName = "type")]
         public string Type { get; set; }
 
-        /// <summary>
```
[the remark this answers is not in the document]
Comment (meiyan-Li) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/Models/StoredProcedureParameter.cs]: Thanks! How to do major version bump? Any documents I can refer?

[the remark this answers is not in the document]
Comment (dsgouda) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/Models/StoredProcedureParameter.cs]: [These](https://semver.org/) are the versioning guidelines we follow.
Essentially the Nuget version must be set to `3.0.0`. Please make sure to update the `AssemblyInfo.cs` file too

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good apart from versioning

File: src/SDKs/DataFactory/Management.DataFactory/Generated/Models/StoredProcedureParameter.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (meiyan-Li) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/Models/StoredProcedureParameter.cs]: Thanks Deepak. Updated to 3.0.0

**Review by dsgouda — APPROVED** [review]
LGTM

**Review by hvermis — APPROVED** [review]

**Comment by hvermis** [comment]
@dsgouda Can you please merge this PR? I will update the release notes once you merge this so I can start our release.

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
 # Changelog for the Azure Data Factory V2 .NET SDK
 
 ## Current version
+## Version 3.0.0
+
+###  Feature Additions
```
[the remark this answers is not in the document]
Comment (hvermis) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: @meiyan-Li I will fix these once @dsgouda merges this PR.

**Pull request merged**